test/kitchen/test/integration/dd-security-agent-stress/rspec/spec_helper.rb

```diff
@@ -0,0 +1,920 @@
+  sleep 5
+end
+
+def stop(flavor)
+  service = get_service_name(flavor)
+  if os == :windows
+    # forces the trace agent (and other dependent services) to stop
+    result = system "net stop /y #{service} 2>&1"
+    sleep 5
+  else
+    if has_systemctl
+      result = system "sudo systemctl stop #{service}.service"
+    elsif has_upstart
+      result = system "sudo initctl stop #{service}"
+    else
+      result = system "sudo /sbin/service #{service} stop"
+    end
+  end
+  wait_until_service_stopped(service)
+  result
+end
+
+def start(flavor)
+  service = get_service_name(flavor)
+  if os == :windows
+    result = system "net start #{service} 2>&1"
+    sleep 5
+  else
+    if has_systemctl
+      result = system "sudo systemctl start #{service}.service"
+    elsif has_upstart
+      result = system "sudo initctl start #{service}"
+    else
+      result = system "sudo /sbin/service #{service} start"
+    end
+  end
+  wait_until_service_started(service)
+  result
+end
+
+def restart(flavor)
+  service = get_service_name(flavor)
+  if os == :windows
+    # forces the trace agent (and other dependent services) to stop
+    if is_service_running?(service)
+      result = system "net stop /y #{service} 2>&1"
+      sleep 5
+      wait_until_service_stopped(service)
+    end
+    result = system "net start #{service} 2>&1"
+    sleep 5
+    wait_until_service_started(service)
+  else
+    if has_systemctl
+      result = system "sudo systemctl restart #{service}.service"
+      # Worst case: the Agent has already stopped and restarted when we check if the process has been stopped
+      # and we lose 5 seconds.
+      wait_until_service_stopped(service, 5)
+      wait_until_service_started(service, 5)
+    elsif has_upstart
+      # initctl can't restart
+      result = system "(sudo initctl restart #{service} || sudo initctl start #{service})"
+      wait_until_service_stopped(service, 5)
+      wait_until_service_started(service, 5)
+    else
+      result = system "sudo /sbin/service #{service} restart"
+      wait_until_service_stopped(service, 5)
+      wait_until_service_started(service, 5)
+    end
+  end
+  result
+end
+
+def has_systemctl
+  system('command -v systemctl 2>&1 > /dev/null')
+end
+
+def has_upstart
+  system('/sbin/init --version 2>&1 | grep -q upstart >/dev/null')
+end
+
+def info
+  `#{agent_command} status 2>&1`
+end
+
+def integration_install(package)
+  `#{agent_command} integration install -r #{package} 2>&1`.tap do |output|
+    raise "Failed to install integrations package '#{package}' - #{output}" unless $? == 0
+  end
+end
+
+def integration_remove(package)
+  `#{agent_command} integration remove -r #{package} 2>&1`.tap do |output|
+    raise "Failed to remove integrations package '#{package}' - #{output}" unless $? == 0
+  end
+end
+
+def integration_freeze
+  `#{agent_command} integration freeze 2>&1`.tap do |output|
+    raise "Failed to get integrations freeze - #{output}" unless $? == 0
+  end
+end
```

Comment:
Does the security agent have specific integrations? I don't understand why we need these.

Reply:
I removed this file